microsoft-edge/web-platform/samples.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I'm not sure if we need this page at all.

None of the web platform APIs that are demo'd on the repo are documented or used on LMC.
These demo pages were not created for LMC, but instead for blog posts or conference talks.

Reply:
removed web-platform/samples.md